assembl/static2/js/app/utils/csrf.js

```diff
@@ -0,0 +1,11 @@
+import request from 'superagent';
```

Comment:
Why not just use `fetch`? It's an extra dependency for only a subset of calls made in non-graphql.

Reply:
You're right!